src/mmw/js/src/analyze/views.js

```diff
@@ -457,6 +461,22 @@ var PointSourceTableView = Marionette.CompositeView.extend({
       }).render().el, { closeButton: false });
     },
 
+    nextPage: function() {
+        if (this.collection.hasNextPage()) {
+            this.collection.getNextPage();
+            this.render();
```

Comment:
Forcing a render was the only way I found to solve an issue that caused the first set of records to essentially be "stuck." The paging would still work as a means to cycle through the records, but the paged records would render below the initial set, no matter how many pages were cycled.